useful workflows/target_workflow_2/target_workflow_2-test.wdl

```diff
@@ -0,0 +1,614 @@
+version 1.0
+
+task mock_task_A {
+  meta {
+    description: "A mock task"
+  }
+
+  input {
+    File     input_file_1
+    File     input_file_2
+    File?    input_file_optional
+    String   input_string_default_1 = "this is a default string"
+    String?  input_string_optional
+    Boolean? input_bool_default = false
+    Int?     input_int_optional
+    String   input_string_default_2 = "my-docker-image.123"
+    String   input_string_default_3 = "this is a second default string"
+  }
+
+  Float declared_float = 0.67
+  Int declared_int = 5
```

Comment:
[not a blocker] is this `declared_int` something like disk space size? Is it necessary for the workflow since it's used in the outputs?

Reply:
ah, if it's not being used in the outputs, it could have been something like "disk space size" in the original task, that just didn't get mapped to anything when I cut away the unnecessary parts...